flocker/node/agents/blockdevice.py

```diff
@@ -944,6 +944,42 @@ def get_device_path(blockdevice_id):
         """
 
 
+class IProfiledBlockDeviceAPI(IBlockDeviceAPI):
+    """
+    An interface for ``IBlockDeviceAPI`` implementers that are capable of
+    creating volumes with a specific profile.
+    """
+
```

Comment:
If it's an interface version number do we want it as an attribute or in the name?

IProfiledBlockDeviceAPIV1 or something.